crates/server/src/store/sqlite/authority.rs

```diff
@@ -452,8 +452,6 @@ impl SqliteAuthority {
     #[cfg_attr(docsrs, doc(cfg(feature = "dnssec")))]
     #[allow(clippy::blocks_in_if_conditions)]
     pub async fn authorize(&self, update_message: &MessageRequest) -> UpdateResult<()> {
-        use tracing::debug;
```

Comment:
Why is this removed? In telio we already convert tracing to logs: https://github.com/NordSecurity/libtelio/blob/9d5105a71645c6d20e951775035f6534fbe1d7e3/src/ffi/mod.rs#L170

Reply:
This is a leftover from the previous commit which I didn't catch until I runned the pipelines in this PR.
This is the [commit](https://github.com/NordSecurity/trust-dns/commit/887bba2af977666d45b78e8b644238892da8df67). It is just a cherry-pick of the original. I wrote it in its description.